src/Graphics/GraphicsResource.cs

```diff
@@ -154,6 +161,12 @@ internal protected virtual bool IsHarmlessToLeakInstance
 			}
 		}
 
+		// This has to return an array because some resources have multiple handles... d'oh!
+		internal virtual GraphicsResourceDisposalHandle[] CreateDisposalHandles() 
+		{
+			return null;
```

Comment:
I thought this could be a throw to make sure all types implement this, but I guess states and other types would just return null anyway... could probably throw in 
```suggestion
			// Only certain GraphicsResource types have pointers to dispose
			return null;
```